inventory/group_vars/opentech-sjc

```diff
@@ -1,7 +1,7 @@
 bastion_clouds:
   - opentech-sjc
 
-bonnyci_logs_apache_server_name: logs.opentechsjc.bonnyci.org
+bonnyci_logs_apache_server_name: logs.bonnyci.org
```

Comment:
I would prefer to keep logs.opentechsjc.bonnyci.org as the server name, and add logs.bonnyci.org as a server alias